readme: availability typo (#49)
Mic92/nix-ld: README.md

## Installation

> nix-ld is now part of nixpkgs and will be available starting with NixOS 22.05. There one can enable it with the following

Reviewer: ```suggestion
nix-ld is part of nixpkgs since NixOS 22.05. There one can enable it with the following
```
